### ISSUE TITLE
Subsequent plan/apply forces global cluster recreation when source cluster's storage_encrypted=true

### ISSUE THREAD

**Issue by marinsalinas** [terraform-providers/terraform-provider-aws#15177]
### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

### Affected Resource(s)

* aws_rds_global_cluster

### Terraform Configuration Files

```hcl
data "aws_kms_alias" "this" {
  name  = "alias/rds"
}


# The source_db_cluster_identifier argument does not currently exist
resource "aws_rds_global_cluster" "global_cluster" {
  # ... other configuration ...
  source_db_cluster_identifier = aws_rds_cluster.example.id
}

resource "aws_rds_cluster" "example" {
  snapshot_identifier = aws_db_cluster_snapshot.example.id

  storage_encrypted = true
  kms_key_id              =  data.aws_kms_alias.this.target_key_arn

  lifecycle {
    ignore_changes = [global_cluster_identifier]
  }
}

```

### Expected Behavior

Subsequent plans should show no changes.

### Actual Behavior

The subsequent plan shows a diff on the `storage_encrypted` attribute since now this attribute is marked as optional, it should be computed as well.

```log
/+ resource "aws_rds_global_cluster" "global_cluster" {
      ...
      - storage_encrypted            = true -> null # forces replacement
    }
```
### Steps to Reproduce

1. `terraform apply`
2. `terraform plan`

### Important Factoids

storage_encrypted should be Computed as well.
https://github.com/terraform-providers/terraform-provider-aws/blob/master/aws/resource_aws_rds_global_cluster.go#L99

**Comment by callaingit** [comment]
I had the same issue, needed to add a lifecycle to ignore changes on storage_encrypted as a work-around.

**Comment by bflad** [comment]
I'm working on similar fixes for other attributes in the RDS resources, so picking this up as part of those efforts.

**Comment by bflad** [comment]
The fix for this has been merged and will release with version 3.13.0 of the Terraform AWS Provider, likely later today. 👍
